src/netaddress.cpp

```diff
@@ -461,6 +479,11 @@ bool CNetAddr::IsValid() const
         return false;
     }
 
+    // Yggdrasil addresses always start with 0x02 or 0x03
+    if (IsYggdrasil() && ((m_addr[0] & 0xfe) != 0x02)) {
```

Comment:
or `0x03`? It is written in the comment, but not in the condition.

Reply:
Probably a false-positive. Everything works well. I was just looking for something, some detail which would fix it for me, but found it to be `discover=1` in the end.

Reply:
Right. `((m_addr[0] & 0xfe) != 0x02))` is the same as `((m_addr[0] != 0x02) && (m_addr[0] != 0x03))